homeassistant/components/timer/__init__.py

```diff
@@ -29,14 +29,23 @@
 ENTITY_ID_FORMAT = DOMAIN + ".{}"
 
 DEFAULT_DURATION = 0
+DEFAULT_RESTORE = False
```

Comment:
This is good. This also makes it not a breaking change. So please remove that part of the description.

Reply:
Yeah, there was a bit of a back and forth on this PR whether it should be a breaking change or not. I wanted to wait to update the description until a decision was made.